### ISSUE TITLE
Remove idle_event from examples/event_handling/idle_and_timeout.py

### ISSUE THREAD

**Issue by dstansby** [matplotlib/matplotlib#8296]
Running the example in `examples/event_handling/idle_and_timeout.py` produces the following deprecation warning:

```
$ pythonw examples/event_handling/idle_and_timeout.py 
/Users/dstansby/matplotlib/lib/matplotlib/cbook/deprecation.py:106: MatplotlibDeprecationWarning: idle_event is only implemented for the wx backend, and will be removed in matplotlib 2.1. Use the animations module instead.
  warnings.warn(message, mplDeprecation, stacklevel=1)
```

As the message says, `idle_event` should be replaced by something from the animations module.

**Comment by NinadBhat** [comment]
I would like to take this. Some questions below
1. If idle_event is replaced with something in animation should this example belong to example/animation after the change.
2. From what I got this example is used to "Demonstrate/test the idle and timeout API" but if idle_event is depreciated should the file just be deleted?

**Comment by anntzer** [comment]
Closed by #8454.
